esp/scm/ws_workunits.ecm

```diff
@@ -2116,10 +2123,49 @@ ESPResponse [exceptions_inline] WUDetailsMetaResponse
     ESParray<string, ScopeType> ScopeTypes;
     ESParray<string, Measure> Measures;
 };
+
+ESPStruct [nil_remove] WUEclDefinitionActionResult
+{
+    string EclDefinition;
+    string Action;
+    string Result;
```

Comment:
@wangkx You should probably also return the wuid if one was created, and the queryid if the query was published.